ops/jujuversion.py

```diff
@@ -112,3 +112,8 @@ def is_dispatch_aware(self) -> bool:
     def has_controller_storage(self) -> bool:
         """Determine whether this juju version supports controller-side storage."""
         return (self.major, self.minor, self.patch) >= (2, 8, 0)
+
+    @property
+    def secrets_supported(self) -> bool:
+        """Determine whether this juju version supports `secrets` feature."""
+        return (self.major, self.minor, self.patch) >= (3, 0, 0)
```

Comment:
Let's make this 3.0.2 as that's the first version with the final version of secrets with the `--refresh` arg and `secret-get-info` separate hook command.

```suggestion
        return (self.major, self.minor, self.patch) >= (3, 0, 2)
```